numpy/core/include/numpy/npy_common.h

```diff
@@ -10,6 +10,25 @@
 #include <npy_config.h>
 #endif
 
+// int*, int64* should be propertly aligned on ARMv7 to avoid bus error
+#if !defined(NPY_STRONG_ALIGNMENT) && defined(__arm__) && !(defined(__aarch64__) || defined(_M_ARM64))
+#define NPY_STRONG_ALIGNMENT 1
+#endif
```

Comment:
@seiko2plus @Qiyu8 Thoughts about this. Do you deal with this elsewhere?

Reply:
yes, there's already an opened pr for it, see #18065